README.md

```diff
@@ -1 +1,36 @@
-# Enumerable
\ No newline at end of file
+# Enumerable
+
```

Comment:
- [ ] [Optional] Please add project description and built with sections to your readme file to make it more professional.

⚠️[Note]: This change is marked as optional because the previous reviewer forgot to mention it So please consider doing it⚠️

Reply:
Thank you :pray: